src/wsdl/elements.ts

```diff
@@ -244,6 +244,13 @@ export class ElementElement extends Element {
           let elem: any = {};
           typeStorage[typeName] = elem;
 
+          if (isMany) {
+            typeElement['maxOccurs'] = this.$maxOccurs
+          }
+          if (Boolean(this.$minOccurs)) {
+            typeElement['minOccurs'] = this.$minOccurs
+          }
```

Comment:
please add a test for these changes

Reply:
@jsdevel 
> please add a test for these changes

Are you referring to minOccurs to add the test?
Sorry, about the minOccurs, it may not be necessary, just to match the maxOccurs.
It's only the maxOccurs part I want to fix.

Reply:
yes.  sorry for the late reply.  please add a test to verify that this works as expected now and in the future.

Reply:
@momo1006 have you solved this issue without this PR?

Reply:
@momo1006 and @jsdevel Is this problem now resolved?

Reply:
Thanks @w666 for the quick response. I created a test for this here https://github.com/vpulim/node-soap/pull/1260/files . I don't know how to add it this PR.

I will attempt a fix for the next few hours.